README.md

```diff
@@ -23,48 +23,73 @@ To set up a conda environment for this project, you can use the provided `enviro
 
    ```bash
    git clone https://github.com/PatrickWeller/PanelPal.git
+    ```
+
 2. Create a new conda environment with the following command:
 
    ```bash
-   conda env create -f environment.yaml
+   conda env create -f env/environment.yaml
+    ```
+
 3. Activate the environment:
 
     ```bash
     conda activate PanelPal
-4. Using pip
+    ```
 
-    ```bash
-    pip install -r requirements.txt
-## Usage
+4. Install PanelPal via pip
 
-Run the panel_pal.py script from your terminal.
-
-    python panel_pal.py
-
-The script will prompt you to enter a panel ID (e.g., R59, r59, or 59). The input will be validated to ensure it matches the correct format.
+    ```bash
+    cd PanelPal
+    pip install .
+    ```
 
-Once a valid panel ID is entered, the tool will fetch information from the PanelApp API and display details about the selected panel.
+## Usage
 
-You will be asked to confirm your selection before proceeding. If confirmed, the tool will generate a BED file with gene locus information.
+### check panel
```

Comment:
Maybe make this as "Check Panel" given the code below explains proper usage with lower case. The upper just looks nicer. Do the same for "generate bed file" below

Reply:
Updated as requested. See commit [e6bc2b5](https://github.com/PatrickWeller/PanelPal/pull/74/commits/e6bc2b5a274e5a8801a56f6b46c11d07b78262b4)